benchmark/client/config.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 PairMesh, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+// Config is the config struct for client side pairbenchmark
+type Config struct {
+	endpoint string
+	port     uint16
+	clients  uint16
+	payload  uint32
+	duration uint16
+}
+
+func NewConfig(
```

Comment:
Add comments for all exported functions.

Reply:
Done